common/src/main/kotlin/no/nav/su/se/bakover/common/JmsConfig.kt

```diff
@@ -23,7 +23,7 @@ data class JmsConfig(
                 channel = it.mqChannel
                 queueManager = it.mqQueueManager
                 transportType = WMQConstants.WMQ_CM_CLIENT
-                setBooleanProperty(WMQConstants.USER_AUTHENTICATION_MQCSP, false)
```

Comment:
wat is?

Reply:
https://www.ibm.com/support/pages/how-specify-userid-and-password-when-connecting-mq